pkg/ensurance/collector/nodelocal/load.go

```diff
@@ -0,0 +1,58 @@
+package nodelocal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/shirou/gopsutil/load"
+	"k8s.io/klog/v2"
+
+	"github.com/gocrane/crane/pkg/common"
+	"github.com/gocrane/crane/pkg/ensurance/collector/types"
+)
+
+const (
+	loadCollectorName = "load"
+)
+
+func init() {
+	registerMetrics(loadCollectorName, []types.MetricName{types.MetricNameCpuLoad1Min, types.MetricNameCpuLoad5Min, types.MetricNameCpuLoad15Min}, NewLoadCollector)
+}
+
+type LoadCollector struct {
+	data map[string][]common.TimeSeries
+}
+
+// NewLoadCollector returns a new Collector exposing kernel/system statistics.
+func NewLoadCollector(_ *NodeLocalContext) (nodeLocalCollector, error) {
+
+	klog.V(2).Infof("NewLoadCollector")
+
+	var data = make(map[string][]common.TimeSeries)
+
+	return &LoadCollector{data: data}, nil
+}
+
+func (l *LoadCollector) collect() (map[string][]common.TimeSeries, error) {
+	var now = time.Now()
+	stat, err := load.Avg()
+	if err != nil {
+		return map[string][]common.TimeSeries{}, err
+	}
+
+	if stat == nil {
+		return map[string][]common.TimeSeries{}, fmt.Errorf("stat is nil")
+	}
+
+	klog.V(6).Infof("LoadCollector collected,1minLoad %v, 5minLoad %v, 15minLoad %v", stat.Load1, stat.Load5, stat.Load15)
```

Comment:
be more careful on the logs
- collected, 1minLoad
add a space after `,`
- 1 min load, 4 min load

Reply:
done~